static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -33,10 +36,13 @@ const cellMeasurer = {
 
 function Breadcrumbs() {
   const {currentTime, replay} = useReplayContext();
+  const organization = useOrganization();
+  const hasPerfTab = organization.features.includes('session-replay-trace-table');
```

Comment:
is this the new feature flag? shouldn't we be hiding the old Perf tab if we have the new feature?

Reply:
i'm re-using the same feature, the "perf" tab is gone though. 

This PR changes static/app/views/replays/detail/layout/focusTabs.tsx so that the tab doesn't appear anymore, but the code is still there. There's some code re-used for this, so it's gonna take a followup PR to cleanup